frontend/src/components/elements/Text/Text.tsx

```diff
@@ -151,23 +159,34 @@ class Text extends React.PureComponent<Props> {
         )
       }
 
+      case TextProto.Format.LATEX: {
```

Comment:
We do we need a special `TextProto.Format.LATEX`?

Why not just make `st.latex` internally call `st.markdown("$$%s$$" % body)`? Seems much simpler and less bug-prone...

Reply:
I implemented `st.latex` before I got `st.markdown` working. I agree it's much simpler that way.